src/OddJobs/ConfigBuilder.hs

```diff
@@ -23,6 +23,8 @@ import GHC.Exts (toList)
 import qualified Data.ByteString as BS
 import UnliftIO (MonadUnliftIO, withRunInIO, bracket, liftIO)
 import qualified System.Log.FastLogger as FLogger
+import GHC.Int (Int64)
```

Comment:
Nitpick but actually it's much better to import Data.Int (Int64).
Aside question: Did you insert this import through HLS?

Reply:
Thanks... didn't know.

HLS is configured on my main project, and I was hacking around on `odd-job` in a `tmp` folder without HLS. So this import was written manually by me.